tests/archive_test.py

```diff
@@ -32,15 +34,25 @@ def test_create_archive(self):
         """should successfully generate an archive"""
         result = uploader.create_archive(tags=[TEST_TAG])
         self.assertEqual(2, result.get("file_count"))
-        result = uploader.create_zip(tags=[TEST_TAG], transformations=[{"width": 0.5}, {"width": 2.0}])
-        self.assertEqual(4, result.get("file_count"))
+        result2 = uploader.create_zip(tags=[TEST_TAG], transformations=[{"width": 0.5}, {"width": 2.0}])
+        self.assertEqual(4, result2.get("file_count"))
+        # clean up
+        try:
+            api.delete_resources([result['public_id'], result2['public_id']], resource_type='raw')
```

Comment:
Resources will not be deleted if the test fail.
Instead of deleting here, use `target_tags` to add TEST_TAG to the resulting resource. Then this [line](https://github.com/cloudinary/pycloudinary/blob/52e909762eee48eb93f68e9751ce29ab06e6d22e/tests/api_test.py#L89) in `teardown` should take care of it.